kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wdt/create-domain-script.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+# Copyright 2018, Oracle Corporation and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# Description:
+#
+#   This script generates a domain by calling the WebLogic Deploy Tool (WDT)
+#   bin/createDomain.sh script.
+#
+#   It first installs WDT using the given download URLs, and then runs it using the
+#   given Oracle home, WDT model file, WDT vars file, etc.
+#
+# Usage:
+#
+#   Export customized values for the input shell environment variables as needed
+#   before calling this script.   See also README.
```

Comment:
I assume this is the README from my wdt branch since that's where this script came from?   If so, I don't see it in this pull request...

Reply:
Yes, as indicated in 2) item in the summary. I have removed the reference to README file since we are going to have different doc.